packages/navigator/src/browser/navigator-preferences.ts

```diff
@@ -34,13 +34,19 @@ Configure glob patterns for excluding files and folders from the navigator. A re
             default: {
                 '**/.git': true
             }
-        }
+        },
+        'navigator.enablePreview': {
```

Comment:
It would be better if we split this preference to 2 preferences as in VS Code:
- `list.openMode` should be defined in core, possible values `singleClick` or `doubleClick`, `singleClick` is default. It should be used in different widgets, not only in the navigator, e.g. in git and git history widgets
- `editor.enablePreview` should be defined in `editor-preview` extension, of `boolean` type, `true` is default. If it is `false` the preview extension should not do anything.

See https://stackoverflow.com/questions/40336583/how-to-disable-preview-file-with-single-click-in-vs-code

Reply:
Also, git and history widgets should pass `preview` as true when they call to the opener service. It could be done as a separate PR.

Reply:
Updated preferences. I will file a follow up PR for the git and history widgets after this has been submitted.